scout/adapter/mongo/variant.py

```diff
@@ -292,6 +292,26 @@ def variant(
 
         return variant_obj
 
+    def case_variant_by_coordinates(self, case_id, category, chrom, chrom_end, start, end):
+        """Return a variant for a case that has the given coordinates
+
+        Args:
+            case_id (str): case id
+            category (str): "sv" or "snv"
+            chrom (str): variant chromosome
+            chrom_end (str): end chromosome
+            start (int): start position
+            end (int): end position
+
+        Returns:
+            variant_obj (Variant): a variant object dictionary
+        """
+        query = dict(case_id=case_id, category=category, chromosome=chrom, position=start, end=end)
```

Comment:
You'll need variant type for this as well to get a boost from the index `caseid_category_chromosome_start_end`!